model/schema/allele.yaml

```diff
@@ -1078,7 +1078,7 @@ slots:
       The chromosome to which a transgene has been mapped. Used for WormBase transgenes
       that have been integrated into the genome and mapped to a chromosome.
     domain: Allele
-    range: Chromosome
+    range: AssemblyComponent
```

Comment:
WB does not have specific assembly information for these locations. They are just roman numerals indicating the chromosome identity. If WB is the only one that will use this property, this would probably be better left as range: Chromosome. If others will use it and want to be able to indicate an assembly then we can discuss an option that doesn't require an assembly.

Reply:
@chris-grove That makes sense, and FlyBase has some data that falls into that category as well. I've reverted the change.